data.tf

```diff
@@ -3,7 +3,7 @@
 locals {
   root_path = split("/", abspath(path.root))
   tf_stack  = join("/", slice(local.root_path, length(local.root_path) - 1, length(local.root_path)))
-  mesh_name = "apps"
+  mesh_name = "apps_v2"
```

Comment:
I'd add a variable for this (e.g. `app_mesh_name`, defaulting to `apps`) instead of using a local var, so we don't need to publish a new version of this module when renaming a mesh